tests/report/CMakeLists.txt

```diff
@@ -8,10 +8,14 @@ if (BUILD_ENCLAVES)
 endif()
```

Comment:
Since we're going to add a new library (as described in @CodeMonkeyLeet 's comment above), I think we should add a new test that invokes the function exposed by that library

Reply:
Also, like we talked about yesterday, we can probably enlist @ionutbalutoiu and his team to help us add a new CI (or nightly) stage for testing these packages!